model/src/main/kotlin/FileFormat.kt

```diff
@@ -64,6 +64,16 @@ enum class FileFormat(val mapper: ObjectMapper, val fileExtension: String, varar
          * Return the [FileFormat] for the given [file], or `null` if there is none.
          */
         fun forFile(file: File): FileFormat = forExtension(file.extension)
+
+        /**
+         * Return a list of all files inside [directory] with known extensions that can be deserialized.
+         */
+        fun findDeserializableFiles(directory: File): List<File> {
+            val allFileExtensions = enumValues<FileFormat>().flatMap { it.fileExtensions }
+            return directory.walkBottomUp().filter {
+                it.isFile && it.extension in allFileExtensions && !it.isHidden
```

Comment:
@fviernau do you guys actually make use of the hidden flag to "disable" files, or could we drop `!it.isHidden` for simplicity?

Reply:
We are not using the "hidden" flag to "disable" package configurations.